README.md

```diff
@@ -16,6 +14,19 @@ We may have private repositories if our collaborators are more comfortable with
 
 If you're new to the lab, check out the [Onboarding repository](https://github.com/WhitakerLab/Onboarding) which will talk you through the different exercises and actions for your first few weeks in the lab.
 
+## Weekly updates
+
+Lab members are asked to prepare for their regular meetings with Kirstie by completing their "Weekly-meetings" file and open a pull request to update the repository in advance of the meeting.
+You can find the public records of these meetings in the [weekly-updates](weekly-updates) folder.
+
+## Lab meetings
+
+Lab meetings are held every week on Thursdays at 3pm.
+Remote attendance is always supported.
+Participation (even if you can't attend the meeting itself) is required from all lab members.
+
```

Comment:
I'd like to suggest  we could add a reminding here just like "Please pay attention and participant to the meeting HackMD in advance"....Because I initially had a misunderstand that HackMD should be filled during the meeting.:smile:

Reply:
Thanks @minnieho1115 - I've added a sentence now. Let me know what you think.

Reply:
Looks good,thank you.:rose: